### PR TITLE
Fix Ace labels do not work with more than 9 tabs

### PR DESCRIPTION
**Pull request by gpanago**
Ref: #231

### CODE REVIEW

**Comment by jcs090218** [comment]
Can you resolve the conflict? Thank you!

**Comment by gpanago** [comment]
Of course, done!

File: centaur-tabs-functions.el [hunk 1]
```diff
@@ -727,17 +727,19 @@ Call `centaur-tabs-tab-label-function' to obtain a label for TAB."
...
-                                   centaur-tabs-ace-jump-keys))
-                 'centaur-tabs-tab tab
-                 'face (if selected-p
-                           'centaur-tabs-jump-identifier-selected
-                         'centaur-tabs-jump-identifier-unselected)
-                 'pointer centaur-tabs-mouse-pointer
-                 'help-echo buf-file-name
-                 'local-map centaur-tabs-default-map)))
+                (when-let ((position (nth (cl-position tab (centaur-tabs-view (centaur-tabs-current-tabset t)))
+                                          centaur-tabs-ace-jump-keys)))
```
Comment (jcs090218) [inline centaur-tabs-functions.el:731]: Should we use `when` here since we are not using the variable `position`? 🤔

Comment (gpanago) [reply centaur-tabs-functions.el:731]: Apologies, fixed now.

**Review by jcs090218 — APPROVED** [review]

**Comment by jcs090218** [comment]
Thank you!

**Pull request merged**